### PR TITLE
Bump stack to 0.1.5.0

### PR DESCRIPTION
**Pull request by wereHamster**

### CODE REVIEW

File: Casks/stack.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-  url "https://github.com/commercialhaskell/stack/releases/download/v#{version}/stack-#{version}-x86_64-osx.gz"
+  url "https://github.com/commercialhaskell/stack/releases/download/v#{version}/stack-#{version}-x86_64-osx.tar.gz"
   appcast 'https://github.com/commercialhaskell/stack/releases.atom'
   name 'stack'
   homepage 'https://github.com/commercialhaskell/stack'
   license :bsd
 
-  binary "stack-#{version}", :target => 'stack'
+  binary "stack-#{version}-x86_64-osx/stack", :target => 'stack'
```
Comment (vitorgalvao) [inline Casks/stack.rb]: You can remove `, :target => 'stack'`. No longer needed.

**Comment by wereHamster** [comment]
Fixed.

**Comment by wereHamster** [comment]
Apparently the `set_permissions` path doesn't exist anymore, and on a second box installing stack fails (because chmod can't find the file). But on my laptop it works without problems. Any idea why?

**Comment by wereHamster** [comment]
Is set_permissions in postflight needed? It works without.

**Comment by jawshooah** [comment]
The target of the `set_permissions` call was the binary file. Since the binary file has changed as of this PR, that target needs to change as well.

**Comment by wereHamster** [comment]
I figured that much. But is `set_permissions` even needed? The file in the tarball already has +x permissions.

**Comment by jawshooah** [comment]
If that's the case, then no.

**Pull request merged**